spec/fixtures/maven/sample_maven_java_spock_project/README.adoc

```diff
@@ -0,0 +1,64 @@
+[.float-group]
```

Comment:
Could this file be removed? It doesn't seem necessary for the specs

Reply:
Hi codeinabox, I will remove such unnecessary files in fixtures.
I just copied and modified it slightly for the tests.
Thanks.